### PR TITLE
Updates based on onboarding feedback

### PR DESCRIPTION
**Pull request by jasonjgeiger**
Modified the enable app section to add the additional step of "installed apps". Also removed the inline CSV, since the file download is available.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/career-coach.md [hunk 1]
```diff
@@ -255,6 +248,10 @@ Career Coach can be customized to be unique to your educational institution. The
   
 3. Select **Apply**.
 
+## Making Career Coach available to your organization
+
+Now that Career Coach has been configured for your organization. Follow these steps to ensure that Career Coach is available to organization in Microsoft Teams.
```
Comment (JohanFreelancer9) [inline Teams/career-coach.md:253]:
```suggestion
Now that Career Coach has been configured for your organization, follow these steps to ensure that Career Coach is available to your organization in Microsoft Teams.
```

File: Teams/career-coach.md [hunk 2]
```diff
@@ -271,21 +268,37 @@ After you complete the configuration, enable the app for students and licensed u
   > [!NOTE]
   > Allowed means that the app is available for users in your educational institution. Blocked means that the app isn't available to students.
 
-#### Pin the app
+### Add Career Coach as an installed app
+
+> [!NOTE]
+> This step ensures 1) that Career Coach is properly configured for your organization 2) that students find Career Coach.
```
Comment (JohanFreelancer9) [inline Teams/career-coach.md:274]:
```suggestion
> This step ensures that Career Coach is properly configured for your organization and that students find Career Coach.
```

File: Teams/career-coach.md [hunk 3]
```diff
@@ -271,21 +268,37 @@ After you complete the configuration, enable the app for students and licensed u
...
+
+1. Sign in to the **Teams admin center**.
+
+2. Select **Teams apps** &gt;**Setup policies** &gt; *Your policy*.
```
Comment (JohanFreelancer9) [inline Teams/career-coach.md:278]:
```suggestion
2. Select **Teams apps** &gt; **Setup policies** &gt; **Your policy**. 
```

File: Teams/career-coach.md [hunk 4]
```diff
@@ -271,21 +268,37 @@ After you complete the configuration, enable the app for students and licensed u
...
+### Add Career Coach as an installed app
+
+> [!NOTE]
+> This step ensures 1) that Career Coach is properly configured for your organization 2) that students find Career Coach.
+
+1. Sign in to the **Teams admin center**.
+
+2. Select **Teams apps** &gt;**Setup policies** &gt; *Your policy*. 
+
+3. Under Installed apps, select Add apps.
```
Comment (JohanFreelancer9) [inline Teams/career-coach.md:280]:
```suggestion
3. Under Installed apps, select Add apps.
```

File: Teams/career-coach.md [hunk 5]
```diff
@@ -271,21 +268,37 @@ After you complete the configuration, enable the app for students and licensed u
...
+
+4. In the Add installed apps pane, search for the apps you want to automatically install for users when they start Teams. You can also filter apps by app permission policy. When you've chosen your list of apps, select Add.
```
Comment (JohanFreelancer9) [inline Teams/career-coach.md:282]:
```suggestion
4. In the Add installed apps pane, search for the apps you want to automatically install for users when they start Teams. You can also filter apps by app permission policy. When you've chosen your list of apps, select **Add**.
```

File: Teams/career-coach.md [hunk 6]
```diff
@@ -271,21 +268,37 @@ After you complete the configuration, enable the app for students and licensed u
...
+
+### Pin the app
 
 Pinning Career Coach will make the app more accessible and visible for students.
 
 1. Sign in to the **Teams admin center**.
 
-1. Select **Teams apps** &gt;**Setup policies** &gt; *Your policy*. 
+2. Select **Teams apps** &gt;**Setup policies** &gt; *Your policy*.
```
Comment (JohanFreelancer9) [inline Teams/career-coach.md:290]:
```suggestion
2. Select **Teams apps** &gt;**Setup policies** &gt; **Your policy**. 
```

File: Teams/career-coach.md [hunk 7]
```diff
@@ -271,21 +268,37 @@ After you complete the configuration, enable the app for students and licensed u
...
 
-1. Search for **Career Coach**, and then select **Add**.
+5. Choose the order for the app to appear and select **Save**.
 
-1. Choose the order for the app to appear and select **Save**.
+> [!NOTE]
+> Students will be notified in Microsoft Teams that Career Coach has been pinned.
 
-   Students will be notified in Microsoft Teams that Career Coach has been pinned.  
+Reference [Manage app setup policies in Microsoft](https://docs.microsoft.com/en-us/microsoftteams/teams-app-setup-policies) for additional details.
```
Comment (JohanFreelancer9) [inline Teams/career-coach.md:301]:
```suggestion
For more information, see [Manage app setup policies in Microsoft](/microsoftteams/teams-app-setup-policies) for additional details.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @jasonjgeiger, kindly commit my suggested changes. Many thanks.

**Pull request merged**